components/builder-jobsrv/src/migrations/mod.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
```

Comment:
The date the file was originally authored is the right pattern for the license. We've got a number of files that have a range because of a commit that was done in early 2017 but I've slowly been setting it back to the single date as I modify files. I probably should just go through the whole codebase and fix it.

Reply:
Sure, easy enough to fix.